### PR TITLE
feature: View map from hotel details page

### PR DESCRIPTION
**Pull request by dave-ok**
Please close and merge #122  before this PR. This is an incremental PR

### CODE REVIEW

File: src/containers/facility/FacilityIntroduction.tsx [hunk 1]
```diff
@@ -146,42 +174,64 @@ export const FacilityIntroduction = ({
...
-    accommodation?.contactInformation?.address?.streetAddress,
-    accommodation?.contactInformation?.address?.locality,
-    accommodation?.contactInformation?.address?.premise,
-    accommodation?.contactInformation?.address?.country
-  ]
-    .filter(Boolean)
-    .join(', ');
+  const address = buildAccommodationAddress(accommodation);
+
+  console.log(largestImagesUrls);
```
Comment (bobetbat) [inline src/containers/facility/FacilityIntroduction.tsx:229]: rm logs

Comment (dave-ok) [reply src/containers/facility/FacilityIntroduction.tsx:229]: Oops! We should probably include a `console: warn` rule in EsLint?

File: package.json [hunk 2]
```diff
@@ -126,5 +126,7 @@
   "resolutions": {
     "autoprefixer": "10.4.5"
   },
-  "dependencies": {}
+  "dependencies": {
+    "react-image-lightbox": "^5.1.4"
```
[the remark this answers is not in the document]
Comment (dave-ok) [reply package.json:130]: Moving...

File: src/components/MapBox.tsx [hunk 3]
```diff
@@ -73,6 +82,10 @@ export const MapBox: React.FC = () => {
   const { selectedFacilityId } = useAppState();
   const dispatch = useAppDispatch();
 
+  // to highlight a given marker use url params "focusedFacilityId"
+  const [searchParams] = useSearchParams();
+  const focusedFacilityId = useMemo(() => searchParams.get("focusedFacilityId"), [searchParams]);
```
[the remark this answers is not in the document]
Comment (bobetbat) [reply src/components/MapBox.tsx]: I believe they serve same purpose, for user to identify between the only one he selects/hovers or redirected. So I would leave one and remove the other?

**Review by bobetbat — APPROVED** [review]
LGTM

**Pull request merged**